### DISCUSSION TITLE
Tree visualization?

### DISCUSSION THREAD

**Discussion by MoonKraken** [smartcorelib/smartcore#229] [Q&A] [answered]
Is there any way to create a visualization of a decision tree such as the one in the docs? https://smartcorelib.org/assets/imgs/tree.svg

I didn't currently see a way to do that but i just wanted to make sure 😎

**Comment by Mec-iS** [comment, marked as answer]
thanks for the question.

No at the moment we don't provide in-library visualizations but you can achieve that by using the `plotters` library as showed at the bottom of this notebook:   https://github.com/smartcorelib/smartcore-jupyter/blob/main/notebooks/03.00-Logistic-Regression.ipynb

I don't know how that diagram was created, if you find out I will be glad to know.
we can consider providing plotting traits in the future when the API will be stable, anyway the use of notebooks looks to be and probably will be in my opinion more convenient.
